src/data/boss.json

```diff
@@ -163,6 +163,98 @@
         "wikiLink": "https://escapefromtarkov.fandom.com/wiki/knight",
         "behavior": "Rush"
     },
+    {
+        "name": "Big Pipe",
+        "normalizedName": "big-pipe",
+        "details": "The leader of 'The Goons'. Can spawn on many different maps.",
+        "health": 910,
+        "loot": [
```

Comment:
The "loot" field in this file isn't currently used for any Mob that has data on SPT. Doesn't hurt to leave it here, but also won't change how the pages are displayed.

Reply:
i got all data from

https://db.sp-tarkov.com/search